application/src/main/java/uk/gov/hmcts/ccd/definition/store/CacheConfiguration.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.ccd.definition.store;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.github.benmanes.caffeine.cache.RemovalCause;
+import com.github.benmanes.caffeine.cache.RemovalListener;
+import java.util.concurrent.TimeUnit;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.caffeine.CaffeineCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@EnableCaching
+@Configuration
+public class CacheConfiguration {
+
+    @Value("${user.cache.ttl.secs}")
+    private Integer userCacheTTLSecs;
+
+    @Bean
+    public CacheManager cacheManager() {
+        CaffeineCacheManager cacheManager = new CaffeineCacheManager("userInfoCache");
```

Comment:
I guess spring boot has already this class with customisation via properties.. 
`CaffeineCacheManager`
https://docs.spring.io/spring-boot/docs/2.1.6.RELEASE/reference/html/boot-features-caching.html#boot-features-caching-provider-caffeine